relay/submit.go

```diff
@@ -245,13 +242,13 @@ func verifyBlock(sbr structs.SubmitBlockRequest, beaconState State) (retry bool,
 		return false, fmt.Errorf("%w: got %d, expected %d", ErrInvalidTimestamp, sbr.Timestamp(), expectedTimestamp)
```

Comment:
I think this may also fail if we check for exact comparison

Reply:
Not really, because timestamp is only calculated based on Genesis + the slot of `submitBlockRequest`